crates/yttrium/src/chain_abstraction/api/prepare.rs

```diff
@@ -96,7 +96,7 @@ impl FundingMetadata {
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[cfg_attr(feature = "uniffi", derive(uniffi_macros::Record))]
 #[serde(rename_all = "camelCase")]
-pub struct RouteResponseAvailable {
+pub struct PrepareResponseAvailable {
```

Comment:
Why are we changing type names?

Reply:
I'm changing it to align more closely to the API it's a part of `prepare`. `route` was the old name